### PR TITLE
DM-47334: Add yaml file to specify targets for comcom sv observations

### PR DESCRIPTION
**Pull request by bechtol**
This PR adds the field_survey_centers.yaml file that was intended to be included in DM-44618.

### CODE REVIEW

File: python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py [hunk 1]
```diff
@@ -131,3 +131,43 @@ def get_scheduler(self) -> typing.Tuple[int, CoreScheduler]:
...
+    -------
+    target_dict : `dict`
+        Dictionary of candidate target names and coordinates.
+    """
+    infile = str(get_data_dir() / "field_survey_centers.yaml")
+    with open(infile) as stream:
+        try:
+            targets_dict = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
```
Comment (tribeiro) [inline python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py:171]: You really want to skip parsing yaml errors? I would suggest you remove this try/except here. Or is there a reason I am missing to do this?

File: python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py [hunk 2]
```diff
@@ -131,3 +131,43 @@ def get_scheduler(self) -> typing.Tuple[int, CoreScheduler]:
...
+    ----------
+    exclude : `list[str]`
+        List of target names to exclude when loading.
+
+    Returns
+    -------
+    target_dict : `dict`
+        Dictionary of candidate target names and coordinates.
+    """
+    infile = str(get_data_dir() / "field_survey_centers.yaml")
```
Comment (tribeiro) [inline python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py:166]: I don't think you need to convert this to a string.

File: python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py [hunk 3]
```diff
@@ -131,3 +131,43 @@ def get_scheduler(self) -> typing.Tuple[int, CoreScheduler]:
         scheduler = CoreScheduler(self.surveys, nside=self.nside)
 
         return self.nside, scheduler
+
+
+def without_keys(d: dict, keys: typing.List[str]) -> dict:
```
Comment (tribeiro) [inline python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py:136]: Do you really need this method at al? it is a one-liner. Just do it inline?

**Review by tribeiro — COMMENTED** [review]
I have some comments for you. 

Also, can you please add a unit test for `get_comcam_sv_targets`?

File: python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py (hunk 1 above)
Comment (bechtol) [reply python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py]: Removed the try/except

File: python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py (hunk 3 above)
Comment (bechtol) [reply python/lsst/ts/fbs/utils/maintel/make_fieldsurvey_scheduler.py]: Going with inline option

**Pull request merged**